config/settings/base.py

```diff
@@ -69,13 +69,10 @@
 # https://docs.djangoproject.com/en/dev/ref/settings/#caches
 CACHES = {
     "default": {
-        "BACKEND": "django_redis.cache.RedisCache",
+        "BACKEND": "redis_lock.django_cache.RedisCache",
```

Comment:
Lets keep the backend as the older one.

Reply:
we need the redis_lock class as it adds the lock method on django_redis cache backend, it makes handling the connection for redis_lock easier


https://python-redis-lock.readthedocs.io/en/latest/readme.html